### PR TITLE
Sustain: show more prominent content age message on old articles

### PR DESCRIPTION
**Pull request by JustinPinner**
We've changed the rules from the original PR (https://github.com/guardian/frontend/pull/10825).

We now only show the old content message on anything 30 days old or more and that is tagged with `tone/news`. And only if the feature is switched on.

The screenshots below show what it looks like (the crystal maze travel story wouldn't usually show up as it isn't tagged appropriately but the appearance will be the same for correctly tagged content - there just wasn't any matching old stuff in the most popular list at the time) 

screen shot 2015-12-02 at 10 55 39
screen shot 2015-12-02 at 10 57 04
screen shot 2015-12-02 at 12 08 05

### CODE REVIEW

**Comment by paperboyo** [comment]
> We now only* show the old content message on anything 30 days old or more and that is tagged with tone/news. And only if the feature is switched on.

* God bless you! Phew!

That said - wasn't there a funny occurrence with an obituary? [I haven't said that. I'm actually against shielding readers from themselves...]

File: common/app/conf/switches/FeatureSwitches.scala [hunk 1]
```diff
@@ -5,6 +5,15 @@ import org.joda.time.LocalDate
 
 trait FeatureSwitches {
 
+  val contentAgeMessageSwitch = Switch(
+    "Feature",
+    "content-age-message",
+    "Show old content message on... old content",
```
Comment (jennysivapalan) [inline common/app/conf/switches/FeatureSwitches.scala]: Should we add here that it's just on tone/news?

File: common/app/views/support/ContentOldAgeDescriber.scala [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  def apply(content: model.Content): String = {
+    message(content.trail.webPublicationDate)
+  }
+
+  def apply(apiContent: com.gu.contentapi.client.model.Content): String = {
+    message(apiContent.webPublicationDate)
+  }
+
+  private def message(pubDate: DateTime) = {
+    val warnLimitDays = 30   // config this!
```
Comment (jennysivapalan) [inline common/app/views/support/ContentOldAgeDescriber.scala]: Not sure it's worth a config but worth removing the comment?

**Comment by jennysivapalan** [comment]
:+1: :older_man: :older_woman:

**Pull request merged**